tests/driver_sx1280/README.md

```diff
@@ -0,0 +1,9 @@
+driver_sx1280
+=============
+
+Describe here what the application is about.
```

Comment:
Can you add some documentation here? You can base yourself on `tests/driver_sx126x/README.md` and on the test procedure you have in the PR description.